<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leaf</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Icons/res/app.jpg</normaloff>:/Icons/res/app.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(248, 248, 248, 255), stop:1 rgba(255, 255, 255, 255));
}

QStatusBar {
    background-color: #33b864;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="content">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="SellingForm" name="sellingPage"/>
      <widget class="OrdersForm" name="ordersPage"/>
      <widget class="ChashForm" name="chashPage"/>
      <widget class="SuppliersForm" name="suppliersPage"/>
      <widget class="ProdectsForm" name="prodectsPage"/>
      <widget class="ClientsForm" name="clientsPage"/>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>48</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #33b864;
padding-left: 24px;
padding-right: 24px;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;Prodects&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QWidget" name="nav" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="sellingBtn">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Selling</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prodectsBtn">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Prodects</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clientsBtn">
         <property name="text">
          <string>Clients</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="suppliersBtn">
         <property name="text">
          <string>Suppliers</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cashBtn">
         <property name="text">
          <string>Cash</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ordersBtn">
         <property name="text">
          <string>Orders</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuStock">
    <property name="title">
     <string>Stock</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAddFamily"/>
    <addaction name="actionAddCategory"/>
    <addaction name="separator"/>
    <addaction name="actionAddProduct"/>
    <addaction name="separator"/>
    <addaction name="actionAddOrder"/>
   </widget>
   <widget class="QMenu" name="menuManagement">
    <property name="title">
     <string>Management</string>
    </property>
    <addaction name="actionAddUser"/>
    <addaction name="separator"/>
    <addaction name="actionAddSupplier"/>
    <addaction name="actionAddClient"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStock"/>
   <addaction name="menuManagement"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddUser"/>
   <addaction name="actionAddProduct"/>
   <addaction name="separator"/>
   <addaction name="actionAddFamily"/>
   <addaction name="actionAddCategory"/>
   <addaction name="actionAddClient"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddProduct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icons/res/trolley.png</normaloff>:/Icons/res/trolley.png</iconset>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </action>
  <action name="actionAddFamily">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icons/res/family-tree.png</normaloff>:/Icons/res/family-tree.png</iconset>
   </property>
   <property name="text">
    <string>Add Family</string>
   </property>
  </action>
  <action name="actionAddCategory">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icons/res/category.png</normaloff>:/Icons/res/category.png</iconset>
   </property>
   <property name="text">
    <string>Add Category</string>
   </property>
  </action>
  <action name="actionAddUser">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icons/res/user.png</normaloff>:/Icons/res/user.png</iconset>
   </property>
   <property name="text">
    <string>Add User</string>
   </property>
  </action>
  <action name="actionAddClient">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Icons/res/customer.png</normaloff>:/Icons/res/customer.png</iconset>
   </property>
   <property name="text">
    <string>Add Client</string>
   </property>
  </action>
  <action name="actionAddSupplier">
   <property name="text">
    <string>Add Supplier</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAddOrder">
   <property name="text">
    <string>Add Order</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProdectsForm</class>
   <extends>QWidget</extends>
   <header location="global">prodectsform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SellingForm</class>
   <extends>QWidget</extends>
   <header location="global">sellingform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClientsForm</class>
   <extends>QWidget</extends>
   <header location="global">clientsform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChashForm</class>
   <extends>QWidget</extends>
   <header location="global">chashform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SuppliersForm</class>
   <extends>QWidget</extends>
   <header location="global">suppliersform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrdersForm</class>
   <extends>QWidget</extends>
   <header location="global">ordersform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
